jbehave-support-core/src/main/java/org/jbehavesupport/core/report/extension/RestXmlReporterExtension.java

```diff
@@ -166,17 +169,7 @@ private String handleMultipart(String body) throws IOException {
     }
 
     private Path getBodyDestinationPath() {
```

Comment:
We can probably get rid of this method alltogether and call fileNameResolver directly from `handleMultipart`